src/core/commandRunner.ts

```diff
@@ -52,6 +52,18 @@ export class CommandRunner implements ICommandRunner {
       // Otherwise, it will resolve with the content of the stdout.
       let execProcess = exec(fullCommand, actualOptions, (error, stdout, stderr) => {
         if (error || stderr !== "") {
+          // If there's no error object and the error stream only contains some warning, keep going.
+          let stderrJSON = JSON.parse(stderr);
+          if (
+            !error &&
+            Object.keys(stderrJSON).length === 1 &&
+            stderrJSON.hasOwnProperty("warnings")
+          ) {
+            resolve(`Completed with warnings:
```

Comment:
```suggestion
            resolve(`Completed with warnings:\n${stderr}\n\n${stdout}`);
```
It would be clearer imo and less potential problems with template strings and multi-line.